mass_mailing_welcome_mail/data/base_automation_data.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <data>
+        <record id="email_template" model="mail.template">
+            <field name="name">Welcome mail - Mailing List</field>
+            <field name="model_id" ref="mass_mailing.model_mail_mass_mailing_contact"/>
+            <field name="use_default_to" eval="True"/>
+            <field name="email_from">${object.company_id.email}</field>
+            <field name="lang">${object.env.context.get("lang")}</field>
+            <field name="subject">Welcome to the mailing list</field>
+            <field name="body_html" type="html">
+                <div>Dear <b>${object.name}</b>,
+                    <br/>
+                    <br/>
+                    <h3>Thanks for signing up four our newsletter!</h3>
+                    <p>
+                        Enjoy a discount code for 20% off your first order
+                    </p><br/>
+                    <p><b>CODE: DISCOUNT20</b></p><br/>
+                    <p>Regards,</p>
+                </div>
+            </field>
+        </record>
+    </data>
+
+    <data>
+        <record id="automated_action" model="base.automation">
+            <field name="name">Send Welcome Mail to new subscribers</field>
+            <field name="model_id" ref="mass_mailing.model_mail_mass_mailing_contact"/>
+            <field name="state">email</field>
+            <field name="trigger">on_create</field>
+            <field name="active" eval="True"/>
+            <field name="template_id" ref="email_template"/>
+            <field name="filter_domain">[["list_ids","=","Newsletter"]]</field>
```

Comment:
Why this filter? By default, it should be in any mass mailing list, and be customized on customer.

Reply:
The same that the previous default text, the subscription is usually to the newsletter and to set a filter for default, easier to modify for the user. I delete it

Reply:
But this doesn't count with translations for example. If there's already a list in data with an XML-ID, use `eval` and `ref` for setting this field with the ID of this mass mailing list by default. That's something reasonable, but not to make the comparison by name.

Reply:
Yes, just was a little help/guide for users, but it's ok, I removed it to avoid doubts.

Reply:
But my question remains unanswered: is there a mailing list defined in data with that name? If so, definitively it's advisable to set it as said, due to Odoo intention is to have that one as newsletter. If not, then put all of them.

Reply:
Yes, it's a default list set on data in mass_mailing

Reply:
OK, then it can be interesting to be added, but I let you to decide. If we add it, then:

- Change module title to reflect this: s/mailing list/newsletter.
- Should the module be renamed to `mass_mailing_newsletter_welcome_mail`?

If not added:

- Change module description to say that is for all mailing list, as right now references the popup only.

Reply:
I think it's useful if it's configured by default a specific list, so I changed name, descriptions and settings for it.